packages/react-native/src/notifier.js

```diff
@@ -97,32 +98,45 @@ const Bugsnag = {
       Bugsnag._client._logger.warn('Bugsnag.start() was called more than once. Ignoring.')
       return Bugsnag._client
     }
+
     if (!isDebuggingRemotely) {
       Bugsnag._client = createClient(opts)
       return Bugsnag._client
-    } else {
-      console.warn(REMOTE_DEBUGGING_WARNING)
-      let initialised = false
-      const stubClient = {}
-      CLIENT_METHODS.reduce((accum, m) => {
-        stubClient[m] = new Proxy(() => {
-          console.warn(
-            `This call to Bugsnag.${m}() is a no-op because remote debugging is enabled and Bugsnag has not yet initialized`
-          )
-        }, {
-          apply: (target, thisArg, argumentsList) => {
-            if (!initialised) return Reflect.apply(target, thisArg, argumentsList)
-            return Reflect.apply(Bugsnag._client[m], Bugsnag._client, argumentsList)
+    }
+
+    console.warn(REMOTE_DEBUGGING_WARNING)
+
+    let initialised = false
+
+    const stubSchema = { ...schema }
+    delete stubSchema.apiKey
+    const stubClient = new Client({
+      ...opts,
+      autoTrackSessions: false,
+      autoDetectErrors: false,
+      enabledBreadcrumbTypes: []
+    }, stubSchema, internalPlugins, { name, version, url })
+
+    CLIENT_METHODS.map((m) => {
```

Comment:
This could be a forEach as we're not using the returned array

```suggestion
    CLIENT_METHODS.forEach((m) => {
```

Reply:
There was another instance of `.map` being used in this way, so I changed both.